lowatt_enedis/__init__.py

```diff
@@ -128,6 +128,10 @@ def get_client(service, cert_file, key_file):
         # verification failure.
         method.location = method.location.replace(b'.erdf.fr', b'.enedis.fr')
 
+        # ConsultationMesuresDetaillees location is also misconfigured
+        if method.location == b'http://www.enedis.fr/sge/b2b/services/consultationmesuresdetaillees/v2.0':
+            method.location = b'https://sge-b2b.enedis.fr/ConsultationMesuresDetaillees/v2.0'
+
```

Comment:
good catch :) I had 403/Forbidden error for a while and thought it was because of misconfiguration wrt our certificate in production, and didn't manage to have any useful feedback from there side.

With you patch, it seems to authenticate properly, though I have "SGT562: Problème : le message de demande est malformé", even while it went fine on homologation server. Does it works for you?

Thank you anyway!

Reply:
ok the error is on my side, grandeurPhysique wasn't specified in my client code, everything is ok now. I will merge your PR and do the cosmetic change myself :)

Reply:
Ok thanks, I was checking again and I can confirm it works on production for me (but not on homologation, which is broken on their side I guess 😅).